compiler/qsc_doc_gen/src/generate_docs/tests.rs

```diff
@@ -27,9 +27,7 @@ fn docs_generation() {
         qsharp.summary: "Returns the number of elements in the input array `a`."
         ---
 
-        # Length function
-
-        Namespace: Microsoft.Quantum.Core
+        # Microsoft.Quantum.Core.Length function
```

Comment:
Why did we change the primary heading to include the full namespace? The main heading reads cleaner when shorter, and including the namespace makes it a little awkward when shortly we want to add another namespace in the docs (e.g. mention it is also available in the `std.` namespace).

Reply:
With the docs popping up in a side pane, making the header really long by including the namespace can be pretty ugly.

<img width="780" alt="image" src="https://github.com/user-attachments/assets/5f3f979b-37e7-4c93-ac43-b75132917cb8">

We can live with it for now to get 1.8 out, but please open a tracking issue to improve by changing it back the way it was for a follow up release.

Reply:
I believe this was the result of the discussion on what to do with the namespaces and package aliases. @swernli , if I'm not mistaken, you were the main proponent of this, but maybe I'm wrong. I'm don't have strong opinion if that should be included in the title or not. But if we don't include fully qualified name in the title, we should include it somewhere else in the description - and that part we couldn't agree on. Any suggestions?

Reply:
> But if we don't include fully qualified name in the title, we should include it somewhere else in the description - and that part we couldn't agree on.

The namespace used to be right underneath it. You can see it was deleted in the diff.

Anyways, as mentioned, just resolve this comment and merge for now. There's some other clean-up in this area and the 'std' addition to figure out, so we'll revisit later. Thanks.

Reply:
> The namespace used to be right underneath it. You can see it was deleted in the diff.

Yes. But now it's not just namespace, it's also the package alias (when it is present). And if you use the same library in two different projects with two different aliases, saying just 'namespace' could be confusing. Now we could say something like 'fully qualified name', but that includes the name as well.

It's actually easy to change this now and still open an issue.

Reply:
I updated the code to include only the item name in the title and fully qualified name in the body of the doc.